components/automate-ui/src/app/pages/data-feed/data-feed.component.ts

```diff
@@ -240,6 +242,19 @@ export class DataFeedComponent implements OnInit, OnDestroy {
             bucket: this.createDataFeedForm.controls['bucketName'].value.trim()
           }
         };
+        testConnectionObservable = this.datafeedRequests.testDestinationForMinio(data);
+          break;
+      }
+      case StorageIntegrationTypes.AMAZON_S3: {
+        const data = {
+          url: 'null',
+          aws: {
+            access_key: this.createDataFeedForm.controls['accessKey'].value.trim(),
+            secret_access_key: this.createDataFeedForm.controls['secretKey'].value.trim(),
+            bucket: this.createDataFeedForm.controls['bucketName'].value.trim(),
+            region: event.region
+          }
+        };
         testConnectionObservable = this.datafeedRequests.testDestinationForMinio(data);
```

Comment:
since we are using the same testDestinationforMinio in case of s3 and minio can we name it to something like testDestinationforStorage

Reply:
done